src/othermbc.md

```diff
@@ -16,22 +16,23 @@ outputs, but this appears not to have been done in licensed games.
 
 ### EMS
 
-PinoBatch learned the game selection protocol for EMS flash carts from
-beware, who in turn learned it from nitro2k01. Take this with a grain of
-salt, as it hasn't been verified on the authentic EMS hardware.
+::: warning
 
-A [header](<#The Cartridge Header>) matching any of the
-following is detected as EMS mapper:
+Take the following with a grain of salt, as it hasn't been verified on authentic EMS hardware.
 
--   Header name is "EMSMENU", NUL-padded
--   Header name is "GB16M", NUL-padded
--   Cartridge type ($0147) = $1B and region ($014A) = $E1
+:::
+
+A [header](<#The Cartridge Header>) matching any of the following is detected as EMS mapper:
+
+- Header name is "EMSMENU", NUL-padded
+- Header name is "GB16M", NUL-padded
+- Cartridge type ($0147) = $1B and region ($014A) = $E1
 
 Registers:
 
-$2000 write: Normal behavior, plus save written value in $2000 latch
-$1000 write: $A5 enables configure mode, $98 disables, and other values have no known effect
-$7000 write while configure mode is on: Copy $2000 latch to OR mask
+- $2000 write: Normal behavior, plus save written value in $2000 latch
+- $1000 write: $A5 enables configure mode, $98 disables, and other values have no known effect
```

Comment:
```suggestion
- $1000 write: $A5 enables configure mode, $98 disables it, and other values have no known effect
```

Also, did we ever decide if the british comma should be consistent through the document?

Reply:
I don't know, I just think it's more natural here because it's how you'd pronounce it. The comma here separates different propositions more than this is an enumeration.